specs/phase0/beacon-chain.md

```diff
@@ -732,11 +732,15 @@ def compute_shuffled_index(index: uint64, index_count: uint64, seed: Bytes32) ->
 
     # Swap or not (https://link.springer.com/content/pdf/10.1007%2F978-3-642-32009-5_1.pdf)
     # See the 'generalized domain' algorithm on page 3
-    for current_round in range(SHUFFLE_ROUND_COUNT):
+    for current_round in map(uint64, range(SHUFFLE_ROUND_COUNT)):
         pivot = bytes_to_int(hash(seed + int_to_bytes(current_round, length=1))[0:8]) % index_count
-        flip = (pivot + index_count - index) % index_count
+        flip = uint64((pivot + index_count - index) % index_count)
         position = max(index, flip)
-        source = hash(seed + int_to_bytes(current_round, length=1) + int_to_bytes(position // 256, length=4))
+        source = hash(
+            seed
+            + int_to_bytes(current_round, length=1)
```

Comment:
Does this count as a limitation to the shuffle-round counts, making its maximum value 255 (inclusive)? Should we change it to a uint8 type? And the `SHUFFLE_ROUND_COUNT` too? I don't think clients actually use `uint64` here at least. E.g. [lighthouse uses `uint8`](https://github.com/sigp/lighthouse/blob/4331834003a6206283c692fe0d6bfbc3c6cfc868/consensus/swap_or_not_shuffle/src/shuffle_list.rs#L86).

Reply:
see my response above